unime/src-tauri/tests/common/assert_state_update.rs

```diff
@@ -76,6 +78,12 @@ pub async fn assert_state_update(
                 ..
             } = expected_state;
 
+            println!(
```

Comment:
If you want to log, you should use macro from the `log` crate (such as `log::info`) instead of a `println!()`, but I'm not sure if it's a good idea to log the entire current and expected state for each single test.

Reply:
Log messages will not show up in `#[test]` context unless you utilize some test crate like [tracing_test](https://crates.io/crates/tracing-test). But I don't think that is really necessary though, since a simple `println` does the job I'd say (so we don't need to add `tracing_test` to all our tests).

If it aids to developer experience I think it is a good addition here.